### ISSUE TITLE
Something wrong with the script ExampleSTIR-GATE.sh

### ISSUE THREAD

**Issue by mele-y** [UCL/STIR-GATE-Connection#84]
there is an error " could not open header file '_GTAE.h33' " when i execute  command ./ExampleSTIR-GATE.sh "test".i am sure STIR install 'bin' and GATE 'bin' in the right 'PATH'. it seems like there are some error with the line 27 in GeneratesSTIRGATEImage.sh

**Comment by robbietuk** [comment]
Have you modified any of the scripts with your own parameter files?

**Comment by mele-y** [comment]
i insert the line `echo $SourceFilenames` as you say,and the output is  `_GATE.h33 _GATE.h33`

**Comment by mele-y** [comment]
i didn't modify any paramters.I just add SGCPATH to my PATH,then execute `./ExampleSTIR-GATE.sh test`in folder VoxelisedSimulation

**Comment by robbietuk** [comment]
https://github.com/UCL/STIR-GATE-Connection/blob/108ffc23ccfd981fc284a553c2d45e40f72f2772/VoxelisedSimulation/SubScripts/GenerateSTIRGATEImages.sh#L27-L28It appears that the `awk` command isnt working as expected

**Comment by KrisThielemans** [comment]
@mele-y what system are you running this on?

If you have gawk, you could try to use that one.

**Comment by mele-y** [comment]
Thanks for your help.I'm using bash on ubuntu 18.04 ,I modified all the scripts to avoid awk command ,now it's working.
I am new to PET  and GATE. I don't know whether the SourceMap that GATE output is useful in image reconstrution

**Comment by robbietuk** [comment]
@mele-y Glad you could get it working. I guess this means that `awk` should be a listed requirement.

 The SourceMap is useful if you want to know the ground truth distribution. There are many applications where this might be useful, e.g. developing new priors for iterative reconstruction. It depends on the type of research.

**Comment by KrisThielemans** [comment]
@mele-y I'd be interested in finding out what your `awk` problems were as I'm amazed it didn't work on Ubuntu.

Potentially you'd consider contributing your modification? Of course, it depends what you replaced it with...

**Comment by mele-y** [comment]
https://github.com/UCL/STIR-GATE-Connection/blob/c57744126e5c0e66cf9a89f04756df06b2342102/VoxelisedSimulation/SubScripts/GenerateSTIRGATEImages.sh#L25-L38
@KrisThielemans I recheck the problem,this may not be a problem with `awk`.it seems the problem occurs in the if-else command，the program always excutes the else part.so i just don't use if-else command as follows:
```
ActivityFilename=my_uniform_cylinder.hv
AttenuationFilename=my_atten_image.hv
generate_image $Activity
generate_image $Attenuation
```

**Comment by robbietuk** [comment]
`awk` is used elsewhere [(for example)](https://github.com/UCL/STIR-GATE-Connection/blob/c57744126e5c0e66cf9a89f04756df06b2342102/VoxelisedSimulation/RunGATE.sh#L41-L81) so I highly doubt that `awk` is the issue.

What does the following output:

```
% Activity=../ExamplePhantoms/STIRparFiles/generate_uniform_cylinder.par
% echo `awk -F:= '/output filename/ { print $2 }' $Activity`.hv
```

On my machine, 

```
my_uniform_cylinder
```

**Comment by mele-y** [comment]
@robbietuk On my machine,the output is `my_uniform_cylinder.hv`

**Comment by KrisThielemans** [comment]
the reason was that the script was calling`/bin/sh` while actually requiring `bash`. I fixes this in #85, and got carried away a bit and fixed some other things there.

Please check
